app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, file_name, file_path = command.split()
```

Comment:
Please, make sure that unpacking would not fail in case `split` returns more than three arguments
```suggestion
    command_name, file_name, file_path, *_ = command.split()
```